CRM/Core/SelectValues.php

```diff
@@ -437,7 +437,7 @@ public static function geoProvider() {
    *
    * @throws \CRM_Core_Exception
    */
-  public function taxDisplayOptions() {
+  public static function taxDisplayOptions() {
```

Comment:
@eileenmcnaughton the new unit test which checks settings pseudoconstants hit this (PHP error non-static method called statically).